certbot-nginx/certbot_nginx/constants.py

```diff
@@ -17,9 +17,21 @@
 """Path to the nginx mod_ssl config file found in the Certbot
 distribution."""
 
-UPDATED_MOD_SSL_CONF_DIGEST = ".updated-mod-ssl-conf-digest.txt"
+UPDATED_MOD_SSL_CONF_DIGEST = ".updated-mod-ssl-conf-nginx-digest.txt"
```

Comment:
I should have noticed this before, but want to know something fun? `mod_ssl` refers to the SSL module in the Apache plugin. The Apache plugin was written first and the Nginx plugin very closely mimics that so the name `mod-ssl` is a bit of a misnomer.

Since we're basically stuck with this name forever (unless we want to do extra work in the future to migrate it), we may want to change it to something like `.updated-nginx-ssl-conf-digest.txt`.

At some point, we should probably change references to `mod_ssl` in the code too, but most weren't created by this PR and I think it makes sense to use consistent naming for those that were.